### DISCUSSION TITLE
How can I export the image that was generated by crabviz?

### DISCUSSION THREAD

**Discussion by palering** [chanhx/crabviz#4] [Q&A] [answered]
As the title.

**Comment by chanhx** [comment]
Export is currently not supported. Exporting SVG with external CSS seems a little bit tricky, and there are no plans to include it in the upcoming formal release, but I think this feature will come soon after the formal release.

**Comment by chanhx** [comment, marked as answer]
Version 0.2.0 is just released. You can now export call graphs by right-clicking on them and selecting "Export Call Graph" in the context menu.
